.mercury/skills/pr-review-flow/SKILL.md

```
@@ -49,17 +49,17 @@ last_validated_at: 2026-04-04T00:00:00.000Z
 
 4. **Request re-review** — after all threads are addressed
    ```bash
-   gh pr comment <PR_NUMBER> --body "@coderabbitai All threads addressed. Please re-review."
+   gh pr comment <PR_NUMBER> --body "/review"
    ```
 
-5. **Merge** — only after CodeRabbit approves (no pending changes)
+5. **Merge** — only after the review bot approves (no pending changes)
    ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)